cmd/backup/hooks.go

```diff
@@ -6,6 +6,8 @@ package main
 import (
 	"fmt"
 	"sort"
+
+	utilites "github.com/offen/docker-volume-backup/internal/utilities"
```

Comment:
If you use the name the package itself is using there is no need to have the mapping.

Reply:
I know, but VS Code kept auto adding it every time I saved the file... I think it was confused about something, weird.

Reply:
It'd be nice if we could still get rid of these.